eng/pipelines/installer/jobs/base-job.yml

```
@@ -127,7 +127,7 @@ jobs:
         $(CommonMSBuildArgs)
         $(OfficialBuildArg)
 
-  - ${{ if eq(parameters.osGroup, 'iOS') }}:
+  - ${{ if or(eq(parameters.osGroup, 'iOS'), eq(parameters.osGroup, 'Android')) }}:
```

Or use [`in`:](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#in)

```suggestion
  - ${{ if in(parameters.osGroup, 'iOS', 'Android') }}:
```

---

thanks!